contrib/rpm/README.md

```diff
@@ -0,0 +1,185 @@
+RPM Spec File Notes
+-------------------
+
+The RPM spec file provided here is for Bitcoin-Core 0.12.0 and builds on CentOS
```

Comment:
Well we do the same for the debian files... I think it can make sense to include them, at least if someone is going to maintain them.